README.md

```diff
@@ -35,14 +35,31 @@ curl -X POST http://kong:8001/routes/{route_id}/plugins \
 
 ## Supported Kong Releases
 
-Kong >= 0.14.1
```

Comment:
does it have breaking change to version 0.14.x?

Reply:
The configuration schema has changed with version 1.0.X https://docs.konghq.com/1.0.x/plugin-development/plugin-configuration/
The modification is down into [kong/plugins/route-by-cookie/schema.lua ](https://github.com/redhoyasa/kong-plugin-route-by-cookie/pull/3/commits/abd12ecf7a91d0df137c2a67d472a7a285dd3fbc#diff-d0c0d6e9d5c9d941e8b4470466efdac09cc2d5e0b63094540cb0006b67939568 )

Reply:
sure 👍